db/migrations/045_member_organization.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:organizations) do
+      primary_key :id
+      timestamptz :created_at, null: false, default: Sequel.function(:now)
+      timestamptz :updated_at
+      text :name, null: false, unique: true
+    end
+
+    create_table(:organization_memberships) do
+      primary_key :id
+      timestamptz :created_at, null: false, default: Sequel.function(:now)
+      timestamptz :updated_at
+
+      foreign_key :member_id, :members, null: false, index: true
```

Comment:
Use a new `index [:member_id, :organization_id], unique: true`. This needs a compound index, there should only be a single membership for a particular member in a particular org.